.dev-deploy/web-staking.Dockerfile

```diff
@@ -1,30 +1,57 @@
 ARG NEXT_PUBLIC_APP_ENV=development
 
-# Base stage
-FROM node:20.11.0 AS release
+FROM node:20-alpine AS base
+
+FROM base AS builder
 ARG NEXT_PUBLIC_APP_ENV
+# Check https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine to understand why libc6-compat might be needed.
+RUN apk add --no-cache libc6-compat
+WORKDIR /app
+
+COPY . .
+
+RUN corepack enable pnpm
+
+RUN pnpm i --filter=@sentry/web-staking --filter=@sentry/ui --config.dedupe-peer-dependents=false
 
-# Set environment variables
 ENV NEXT_TELEMETRY_DISABLED=1
 ENV NEXT_PUBLIC_APP_ENV=development
```

Comment:
Is it beneficial to update this and line 30 to reference the ARG set in line 1? `ENV NEXT_PUBLIC_APP_ENV=${NEXT_PUBLIC_APP_ENV}`

Reply:
I only see the downside of having to update the docker compose and the release for a variable that will ALWAYS HAVE TO BE `development` for the PR environments.

Let me know what you think or if this is to be considered a requested change.